src/main/resources/templates/db_scripts/migration_scripts/update_transformations_mapping_profile.sql

```diff
@@ -0,0 +1,127 @@
+UPDATE ${myuniversity}_${mymodule}.mapping_profiles as profiles
+SET jsonb = jsonb_set(jsonb, '{transformations}', (
+  SELECT jsonb_agg(jsonb_set(transformations.value, '{fieldId}', to_jsonb(LOWER(transformations.value ->> 'fieldId'))))
+  FROM ${myuniversity}_${mymodule}.mapping_profiles, jsonb_array_elements(jsonb -> 'transformations') AS transformations
+  WHERE ${myuniversity}_${mymodule}.mapping_profiles.id = profiles.id
+  GROUP BY id
+))
+WHERE profiles.jsonb ->> 'transformations' IS NOT null AND jsonb_array_length(profiles.jsonb -> 'transformations') > 0;
+
+--a Update path fields in mapping profile transformations for item record type - add holdings[*] at the beginning
+UPDATE diku_mod_data_export.mapping_profiles as profiles
+SET jsonb = jsonb_set(jsonb, '{transformations}', (
+  SELECT jsonb_agg(jsonb_set(transformations.value, '{path}', to_jsonb(
+    CASE
+       WHEN transformations.value ->> 'recordType' = 'ITEM' AND transformations.value ->> 'path' NOT LIKE '%holdings[*]%'
+       THEN regexp_replace(transformations.value ->> 'path', '$.', '$.holdings[*].', 'q')
+       ELSE transformations.value ->> 'path'
+    END
+    )))
+  FROM ${myuniversity}_${mymodule}.mapping_profiles, jsonb_array_elements(jsonb -> 'transformations') AS transformations
+  WHERE ${myuniversity}_${mymodule}.mapping_profiles.id = profiles.id
+  GROUP BY id
+))
+WHERE profiles.jsonb ->> 'transformations' IS NOT null AND jsonb_array_length(profiles.jsonb -> 'transformations') > 0;
+
+--a Update fieldIds with default
+UPDATE ${myuniversity}_${mymodule}.mapping_profiles as profiles
+SET jsonb = jsonb_set(jsonb, '{transformations}', (
+  SELECT jsonb_agg(jsonb_set(transformations.value, '{fieldId}', to_jsonb(
+    CASE
+       WHEN transformations.value ->> 'fieldId' LIKE '%linktext'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'linktext', 'linktext.default')
+	   	 WHEN transformations.value ->> 'fieldId' LIKE '%uri'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'uri', 'uri.default')
+	  	 WHEN transformations.value ->> 'fieldId' LIKE '%materialsspecification'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'materialsspecification', 'materialsspecification.default')
+	  	 WHEN transformations.value ->> 'fieldId' LIKE '%materialsspecification'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'materialsspecification', 'materialsspecification.default')
+	  	 WHEN transformations.value ->> 'fieldId' LIKE '%materialsspecification'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'materialsspecification', 'materialsspecification.default')
+	   	 WHEN transformations.value ->> 'fieldId' LIKE '%publicnote'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'publicnote', 'publicnote.default')
+	 	   WHEN transformations.value ->> 'fieldId' LIKE '%publicnote'
+       THEN regexp_replace(transformations.value ->> 'fieldId', 'publicnote', 'publicnote.default')
```

Comment:
same here public note is present 2 times

Reply:
@KVupp thank you, updated